src/markdown-pages/docs/ecosystem/ecosystem.md

```diff
@@ -3,19 +3,19 @@ path: "/docs/ecosystem"
 title: "The Galasa Ecosystem"
 ---
 
-To realise the full power of Galasa, you need to run your tests inside a Galasa ecosystem. The ecosystem enables you to run automated testing away from your workstation. 
+To realise the full power of Galasa, you need to run your tests inside a Galasa ecosystem. The ecosystem enables you to run automated testing away from your workstation, outside of a local JVM. 
 
-The SimBank tests showcase how the Galasa framework can run inside a locally hosted JVM. Local runs use their own Galasa property files to act as configurational services, making it easy to instantiate and run tests. 
+The SimBank tests showcase how the Galasa framework can run inside a locally hosted JVM. Local runs use their own Galasa property files to act as configurational services, making it easy to instantiate and run tests.
 
 However, there are limitations to taking a local-only approach:
 
 - Configuration settings, test results and test artifacts are stored locally, so cannot be shared across teams
```

Comment:
so cannot easily be shared across teams ?
or
so cannot be shared across teams using Galasa tools.

My point is that a local run can be zipped-up and sent to someone via email/slack..etc.

ie: It's possible, but not so easy perhaps.

Reply:
Oh I see, yes will edit that